.gitignore

```diff
@@ -42,3 +43,4 @@ wasapi-downloader.zip
 /wasapi-downloader
 config/settings.local.yml
 coverage/*
+/tmp/*
```

Comment:
If we ignore this, how will it be created in a new deploy?

Reply:
This is a duplicate of line 11, so please remove.

Reply:
Meant to put this in .dockerignore, not .gitignore.